File: MWR/Source/MWRSolution/MWRDBModule/DBModule/TblSysParameter.cs
using System;
using System.Collections.Generic;
using System.Text;
using ComLib.db;

namespace YRKJ.MWR
{

    public class TblSysParameter : BaseDataModule
    {

        private static string TableName = "SysParameter";
        public TblSysParameter()
        {
        }
        public static string getFormatTableName()
        {
            return SqlCommonFn.FormatSqlTableNameString(TableName);
        }

        public static DataColumnInfo[] Columns = 
                new DataColumnInfo[]{
            new DataColumnInfo(true,false,false,false,"ParameterName",SqlCommonFn.DataColumnType.STRING,128),
            new DataColumnInfo(false,true,false,false,"ParameterValue",SqlCommonFn.DataColumnType.STRING,128),
            new DataColumnInfo(false,true,false,false,"Remark",SqlCommonFn.DataColumnType.STRING,45)
        };

        public static DataColumnInfo getParameterNameColumn()
        {
            return Columns[0];
        }
        public static DataColumnInfo getParameterValueColumn()
        {
            return Columns[1];
        }
        public static DataColumnInfo getRemarkColumn()
        {
            return Columns[2];
        }

        private string _ParameterName = "";
        private string _ParameterValue = "";
        private string _Remark = "";

        public string ParameterName
        {
            get
            {
                return _ParameterName;
            }
            set
            {
                _ParameterName = value;
            }
        }
        public string ParameterValue
        {
            get
            {
                return _ParameterValue;
            }
            set
            {
                _ParameterValue = value;
            }
        }
        public string Remark
        {
            get
            {
                return _Remark;
            }
            set
            {
                _Remark = value;
            }
        }



    }
}

File: DemoApp/TblModel/TblSysParameter.java
package com.yrkj.ar.bean;
import com.yrkj.lib.db.BaseDataModule;
import com.yrkj.lib.db.DataColumnInfo;
import com.yrkj.lib.db.SqlCommonFn;
import com.yrkj.lib.db.SqlCommonFn.DataColumnType;

public class TblSysParameter extends BaseDataModule {
    private static String TableName = "SysParameter";
    public TblSysParameter(){
    }
    public static String getFormatTableName(){
        return SqlCommonFn.FormatSqlTableNameString(TableName);
    }

    public static DataColumnInfo[] Columns = 
            new DataColumnInfo[]{
        new DataColumnInfo(true,false,false,false,"ParameterName",DataColumnType.STRING,128),
        new DataColumnInfo(false,true,false,false,"ParameterValue",DataColumnType.STRING,128),
        new DataColumnInfo(false,true,false,false,"Remark",DataColumnType.STRING,45)
    };

    public static DataColumnInfo getParameterNameColumn(){
        return Columns[0];
    }
    public static DataColumnInfo getParameterValueColumn(){
        return Columns[1];
    }
    public static DataColumnInfo getRemarkColumn(){
        return Columns[2];
    }

    public String ParameterName = "";
    public String ParameterValue = "";
    public String Remark = "";

    public String getParameterName() {
        return  ParameterName;
    }
    public void setParameterName(String ParameterName) {
        this.ParameterName = ParameterName;
    }
    public String getParameterValue() {
        return  ParameterValue;
    }
    public void setParameterValue(String ParameterValue) {
        this.ParameterValue = ParameterValue;
    }
    public String getRemark() {
        return  Remark;
    }
    public void setRemark(String Remark) {
        this.Remark = Remark;
    }



}
